src/main/java/org/apache/commons/lang3/CharUtils.java

```diff
@@ -219,7 +221,7 @@ public static char toChar(final String str, final char defaultValue) {
      */
     public static int toIntValue(final char ch) {
         if (!isAsciiNumeric(ch)) {
-            throw new IllegalArgumentException("The character " + ch + " is not in the range '0' - '9'");
+            throw IllegalArgumentExceptions.format("The character %s is not in the range '0' - '9'", ch);
```

Comment:
Shouldn't %s be %c instead? The same applies to some other exceptions using char.

Reply:
I could be indeed but IIRC it does not matter since the result will be the same. Unless something interesting would happen related to Unicode.